app/src/main/java/com/example/habitsmasher/User.java

```diff
@@ -2,6 +2,7 @@
 
 import android.media.Image;
 
+import java.io.Serializable;
```

Comment:
Why's this necessary?

Reply:
I was previously planning to make the User object be passed into the other fragments instead of just the userId/usernames but decided aganist later. I guess I just forgot to delete the import statement from earlier.